packages/react-native-win32/src/Libraries/Components/View/ViewWin32.Props.ts

```diff
@@ -191,6 +191,7 @@ export interface IViewWin32Props extends Omit<RN.ViewProps, ViewWin32OmitTypes>,
    * Note: If typeID is 'Unknown', a typeName must be provided.
    */
   accessibilityAnnotation?: AccessibilityAnnotationInfo;
+  accessibilityDescription?: string;
```

Comment:
For my own understanding, what is the difference between this and `accessibilityHint`? Could we document this in code here?

Reply:
accessibilityHint provides what will happen when they perform an action while accessibilityDescription provides more detailed info specific to the element, like full file location or last editted date etc. I'll document this in code